core-customize/hybris/config/localextensions.xml

```diff
@@ -4,36 +4,27 @@
     <!--<path dir='${HYBRIS_BIN_DIR}/modules-cep'  autoload='false' />-->
     
     <path dir='${HYBRIS_BIN_DIR}' autoload='false' />
-	  <extension name= 'yb2bacceleratorstorefront' />
+	  <extension name= 'spartacussampledata' />
```

Comment:
I recall and did a deployment using this branch with p8. I might have just deployed the js-storefront part, but the sample data for spartacussampledata should be 'bundled' under the custom header in line 251 of the file.

Will make changes and add the sampledata once the deployment is successful and works accordingly